samples/subsys/ipc/rpmsg_service/remote/sample.yaml

```diff
@@ -12,3 +10,4 @@ tests:
     integration_platforms:
       - v2m_musca_b1_ns
```

Comment:
Is this really needed if both applications are now built using sysbuild?

Reply:
Maybe, I don't know well enough this part to comment.